src/components/Organisms/AlertManager/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { useAlertHandlerContext } from "../../../contexts/alert_handler";
+import Alert from "../../Atoms/Alert";
+import React, { useEffect } from "react";
+
+const AlertManager = () => {
+  const AlertHandlerContext = useAlertHandlerContext();
+  useEffect(() => {
```

Comment:
```suggestion
const AlertManager = () => {
  const alertHandlerContext = useAlertHandlerContext();
  useEffect(() => {
```
`useContext` の返り値は単なるオブジェクトなので、ローワーキャメルケースが適切かと思います。

Reply:
@higashiji2000 
対応しました!

Reply:
コードの他の部分がアッパーキャメルケースのままのようで、エラーが出ています。
修正お願いします！